pkg/cmd/admin/policy/reconcile_clusterroles.go

```diff
@@ -88,49 +83,61 @@ func NewCmdReconcileClusterRoles(name, fullName string, f *clientcmd.Factory, ou
 	return cmd
 }
 
-func (o *reconcileClusterOptions) Complete(cmd *cobra.Command, f *clientcmd.Factory, args []string) error {
-	if len(args) != 0 {
-		return errors.New("No arguments are allowed")
-	}
-
+func (o *reconcileClusterOptions) Complete(cmd *cobra.Command, f *clientcmd.Factory) error {
 	oclient, _, err := f.Clients()
 	if err != nil {
 		return err
 	}
 	o.RoleClient = oclient.ClusterRoles()
 
+	o.Output = kcmdutil.GetFlagString(cmd, "output")
+
 	return nil
 }
 
-func (o *reconcileClusterOptions) ReplaceChangedRoles(changedRoles []*authorizationapi.ClusterRole) error {
-	for i := range changedRoles {
-		role, err := o.RoleClient.Get(changedRoles[i].Name)
-		if err != nil && !kapierrors.IsNotFound(err) {
-			return err
-		}
+func (o *reconcileClusterOptions) Validate(args []string) error {
```

Comment:
I'd prefer to keep `Validate` about validating the `Options` object, not the arguments.  If the arguments to the command are wrong, then I think it's reasonable for the `Complete` method to fail because it can't reliably fill in the `Options` object with invalid input.